Frontend/android-native/app/src/main/AndroidManifest.xml

```diff
@@ -152,7 +170,7 @@
         <service
             android:name=".utils.MyFirebaseMessagingService"
             android:largeHeap= "true"
-            android:exported="false">
+            android:exported="true">
```

Comment:
why we need to export this service ?

Reply:
We need to configure our app to work with multiple push notification providers

Reply:
okay